### PR TITLE
[RFR] Launch a new game

### PR DESCRIPTION
**Pull request by HALLERPierre**
[trello](https://trello.com/c/J0Q0i62A/1-1-as-fred-i-want-to-launch-a-new-game)

# TODO

- [x] Create app
- [x] Create landing page
- [x] Create game page
- [x] Add styles
- [x] Add statics files
- [x] Add docker-compose file
- [x] Add Makefile
- [x] Add tests

# Result
empty-board

### CODE REVIEW

File: src/pages/_document.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import Document from "next/document";
```
Comment (HALLERPierre) [inline src/pages/_document.js]: The `src/pages/_document.js` file is necessary to inject styles in the page for `styled-components`. Found this solution in Next.js doc.

File: src/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "next.js"
+  ],
+  "author": "Pierre h",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/marmelab/quixo-isomorphic/issues"
+  },
+  "homepage": "https://github.com/marmelab/quixo-isomorphic#readme",
+  "dependencies": {
+    "babel-plugin-styled-components": "1.10.2",
```
Comment (zyhou) [inline src/package.json]: Babel plugin is dev dep

File: src/pages/game.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  display: inline-block;
+`;
+
+const CubeImage = styled.img`
+  width: 100px;
+  height: 100px;
+`;
+
+const Game = ({ board }) => (
+  <div>
```
Comment (zyhou) [inline src/pages/game.js]: div is needed ?

**Review by zyhou — CHANGES_REQUESTED** [review]
Missing :

- make stop
- readme update
- licence update 
- linting

travis not working ?

**Comment by Kmaschta** [comment]
Also, can you install some linter rules and enforce them?
Would be awesome to have the prettier rules from eslint, this is the one I'm used to.

You can use this template if needed: https://github.com/marmelab/renard/tree/master/linting

**Review by zyhou — APPROVED** [review]

**Review by Kmaschta — APPROVED** [review]

**Pull request merged**